src/main/java/uk/gov/ida/verifyserviceprovider/configuration/ConfigurationConstants.java

```diff
@@ -38,16 +35,41 @@
     String PRODUCTION_METADATASOURCE_URI = PRODUCTION_HOST + METADATASOURCE_PATH;
     String PRODUCTION_HUBCONNECTOR_ENTITY_ID = PRODUCTION_HOST + HUBCONNECTOR_ENTITY_ID_PATH;
 
+    String[] PRODUCTION_ACCEPTABLE_HUBCONNECTOR_ENTITY_IDS = new String[] {
+        "https://www.signin.service.gov.uk/SAML2/metadata/connector",               // (AWS)
+        "https://connector-node.london.verify.govsvc.uk/ConnectorMetadata",         // (GSP)
+        "https://eidas.signin.service.gov.uk/ConnectorMetadata",                    // (potential new #1)
+        "https://connector.eidas.signin.service.gov.uk/ConnectorMetadata",          // (potential new #2)
+        "https://connector-node.eidas.signin.service.gov.uk/ConnectorMetadata",     // (potential new #3)
```

Comment:
After seeing this on the [MSA PR](https://github.com/alphagov/verify-matching-service-adapter/pull/142) I'm wondering if we should put these in `saml-libs` (`IdaConstants` for instance).

Reply:
Have also discussed with Karol and I'll add it to the related tech debt story for this so we can make a decision later (once the entity ids are finalised, and we've got this thing out the door). As you mentioned, `saml-libs` is good to share things, but risks becoming an overgrown do-everything-be-everything, and I wonder if there's a better/more appropriate way to share fixed config values like this.